nodejs/README.md

```diff
@@ -30,24 +30,30 @@ data feed from a particular URL, parsing it as a FeedMessage (the root type of
 the GTFS-realtime schema), and iterating over the results.
 
 ```javascript
-var GtfsRealtimeBindings = require('gtfs-realtime-bindings');
-var request = require('request');
-
-var requestSettings = {
-  method: 'GET',
-  url: 'URL OF YOUR GTFS-REALTIME SOURCE GOES HERE',
-  encoding: null
-};
-request(requestSettings, function (error, response, body) {
-  if (!error && response.statusCode == 200) {
-    var feed = GtfsRealtimeBindings.transit_realtime.FeedMessage.decode(body);
-    feed.entity.forEach(function(entity) {
+import GtfsRealtimeBindings from "gtfs-realtime-bindings";
+import fetch from "node-fetch";
```

Comment:
@gauthier-th i'm overall okay with this, but a couple of small, non-blocking nits that I realized:
- `node-fetch` is a separate NPM package that needs to be installed for this example to work
- someone following this example MUST switch from `type: commonjs` to `type: module` due to `node-fetch`

you may want to consider including those instructions somewhere in the documentation leading up to this example to head off any possible confusion. part of me feels like it would be nice to have a true vanilla "drop-in" example with no outside-of-node packages required, but I don't have super strong feelings about it

Reply:
Note that AFAIK there is currently no API that works out-of-the-box on all platforms (browser, Node.js, Deno, react-native, FaaS platforms). The best contender for this is in fact the Fetch API.

The node-fetch docs say the following though, so we could use `node-fetch@^2`:

> If you cannot switch to ESM, please use v2 which remains compatible with CommonJS. Critical bug fixes will continue to be published for v2.

Reply:
I also hesitated between ESM and CJS, but I saw that ESM is the standard in the [Node.js documentation](https://nodejs.org/api/esm.html#modules-ecmascript-modules) so I wrote it as ESM.

Do you think it would be better to use CJS instead of ESM and add a note that says to install node-fetch v2?

Reply:
I don't have a strong preference as long as we lean towards having overly verbose instructions on how to get a working example!